[Bug]: make test-unit fails with "TypeError: Cannot subclass typing.Any" on Python 3.10

### Check for existing issues

- [x] I have searched the existing issues and checked that my issue is not a duplicate.

### What happened?

I cloned the repository and attempted to run the unit tests using make test-unit. However, the execution failed with a TypeError related to subclassing typing.Any.

Environment:
Python 3.10 
OS: Linux (Ubuntu)

`============================================================== ERRORS ==============================================================
_______________________________ ERROR collecting tests/test_litellm/containers/test_container_api.py _______________________________
tests/test_litellm/containers/test_container_api.py:26: in <module>
    from litellm.llms.openai.containers.transformation import OpenAIContainerConfig
litellm/llms/openai/containers/transformation.py:36: in <module>
    class OpenAIContainerConfig(BaseContainerConfig):
/usr/lib/python3.10/typing.py:378: in __mro_entries__
    raise TypeError(f"Cannot subclass {self!r}")
E   TypeError: Cannot subclass typing.Any
_______________________________ ERROR collecting tests/test_litellm/containers/test_container_api.py _______________________________
tests/test_litellm/containers/test_container_api.py:26: in <module>
    from litellm.llms.openai.containers.transformation import OpenAIContainerConfig
litellm/llms/openai/containers/transformation.py:36: in <module>
    class OpenAIContainerConfig(BaseContainerConfig):
/usr/lib/python3.10/typing.py:378: in __mro_entries__
    raise TypeError(f"Cannot subclass {self!r}")
E   TypeError: Cannot subclass typing.Any
_______________________________ ERROR collecting tests/test_litellm/containers/test_container_api.py _______________________________
tests/test_litellm/containers/test_container_api.py:26: in <module>
    from litellm.llms.openai.containers.transformation import OpenAIContainerConfig
litellm/llms/openai/containers/transformation.py:36: in <module>
    class OpenAIContainerConfig(BaseContainerConfig):
/usr/lib/python3.10/typing.py:378: in __mro_entries__
    raise TypeError(f"Cannot subclass {self!r}")
E   TypeError: Cannot subclass typing.Any
_______________________________ ERROR collecting tests/test_litellm/containers/test_container_api.py _______________________________
tests/test_litellm/containers/test_container_api.py:26: in <module>
    from litellm.llms.openai.containers.transformation import OpenAIContainerConfig
litellm/llms/openai/containers/transformation.py:36: in <module>
    class OpenAIContainerConfig(BaseContainerConfig):
/usr/lib/python3.10/typing.py:378: in __mro_entries__
    raise TypeError(f"Cannot subclass {self!r}")
E   TypeError: Cannot subclass typing.Any
===================================================== short test summary info ======================================================
ERROR tests/test_litellm/containers/test_container_api.py - TypeError: Cannot subclass typing.Any
ERROR tests/test_litellm/containers/test_container_api.py - TypeError: Cannot subclass typing.Any
ERROR tests/test_litellm/containers/test_container_api.py - TypeError: Cannot subclass typing.Any
ERROR tests/test_litellm/containers/test_container_api.py - TypeError: Cannot subclass typing.Any
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 4 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! xdist.dsession.Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=================================================== 4 errors in 86.06s (0:01:26) ===================================================
make: *** [Makefile:85: test-unit] Error 2`


It appears that BaseContainerConfig might be resolving to Any incorrectly in this environment, causing the inheritance issue.

### Steps to Reproduce

1. Clone the repository and navigate to the project directory
2. Run the development setup and test workflow in a fresh environment
`
make install-dev    # Install development dependencies
make format         # Format your code
make lint           # Run all linting checks
make test-unit      # Run unit tests
`


### Relevant log output

```shell

```

### What part of LiteLLM is this about?

Docs

### What LiteLLM version are you on ?

v1.81.3

### Twitter / LinkedIn details

_No response_

--- a/litellm/llms/openai/containers/transformation.py
+++ b/litellm/llms/openai/containers/transformation.py
@@ -29,7 +29,7 @@ if TYPE_CHECKING:
     BaseLLMException = _BaseLLMException
 else:
     LiteLLMLoggingObj = Any
-    BaseContainerConfig = Any
+    from ...base_llm.containers.transformation import BaseContainerConfig
     BaseLLMException = Any
 
 

--- a/litellm/llms/openai/containers/transformation.py
+++ b/litellm/llms/openai/containers/transformation.py
@@ -16,20 +16,17 @@ from litellm.types.containers.main import (
 )
 from litellm.types.router import GenericLiteLLMParams
 
+from ...base_llm.containers.transformation import BaseContainerConfig
+
 if TYPE_CHECKING:
     from litellm.litellm_core_utils.litellm_logging import Logging as _LiteLLMLoggingObj
 
     from ...base_llm.chat.transformation import BaseLLMException as _BaseLLMException
-    from ...base_llm.containers.transformation import (
-        BaseContainerConfig as _BaseContainerConfig,
-    )
 
     LiteLLMLoggingObj = _LiteLLMLoggingObj
-    BaseContainerConfig = _BaseContainerConfig
     BaseLLMException = _BaseLLMException
 else:
     LiteLLMLoggingObj = Any
-    from ...base_llm.containers.transformation import BaseContainerConfig
     BaseLLMException = Any
 
 

